### PR TITLE
Template: fix more pipeline name placeholder errors

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ### Template
 
+- more update ([#2786](https://github.com/nf-core/tools/pull/2786))
```
Comment (maxulysse) [inline CHANGELOG.md:7]: I feel like we're missing changelog from https://github.com/nf-core/tools/pull/2785

Comment (mashehu) [reply CHANGELOG.md:7]: yes, because it was open before we merged #2783

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 ### Template
 
+-  Rename Nextflow tower badge, fix pipeline name and white space errors in template  ([#2785](https://github.com/nf-core/tools/pull/2785)) and  ([#2786](https://github.com/nf-core/tools/pull/2786))
```
Comment (mashehu) [inline CHANGELOG.md:7]:
```suggestion
- Rename Nextflow tower badge, fix pipeline name and white space errors in template  ([#2785](https://github.com/nf-core/tools/pull/2785)) and  ([#2786](https://github.com/nf-core/tools/pull/2786))
```

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 ### Template
 
+- Rename Nextflow tower badge, fix pipeline name and white space errors in template ([#2785](https://github.com/nf-core/tools/pull/2785)) and ([#2786](https://github.com/nf-core/tools/pull/2786))
```
Comment (maxulysse) [inline CHANGELOG.md:7]:
```suggestion
- Fix pipeline name for multiqc yaml versions file ([#2786](https://github.com/nf-core/tools/pull/2786))
```

**Pull request closed without merging**